google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -23,6 +23,7 @@ plugin.compatibility.error.update.link=Please {0}click here{1} to ensure that yo
 
 appengine.name=App Engine (BETA)
 appengine.environment.name.flexible=App Engine flexible
+appengine.environment.name.mvm=App Engine MVM
```

Comment:
I'd agree in general, but see the screenshot attached to the issue. The idea was to match this.